### ISSUE TITLE
[Booking] Impossible to make periodic bookings

### ISSUE THREAD

**Issue by gaetanHerve** [bgo-bioimagerie/platformmanager#198]
This issue (only ?) lies in use of EcResponsible class, which doesn't exist anymore, in BookingperiodicController.php ~line157

**Comment by osallou** [comment]
not single place....

seems should map to core_j_user_responsible/id_resp

**Comment by osallou** [comment]
in migration, it takes ec_responsibles and sets  ClClientUser. So equivalent should be in clients.

**Comment by gaetanHerve** [comment]
Found the fix in "remove module ecosystem replaced by module user" commit on branch master :
We should use
```
$modelResp = new ClClientUser();
$userResps = $modelResp->getUserClientAccounts($recipient_id, $id_space);
```
instead of 
```
$modelResp = new EcResponsible();
$userResps = $modelResp->getUserResponsibles($recipient_id);
```

I fix this in issue#189 merge request

**Comment by osallou** [comment]
fixed in #189
